mango/container/mp.py

```diff
@@ -510,6 +530,18 @@ def handle_message_in_sp(self, message, receiver_id, priority, meta):
         else:
             sp_queue_of_agent.put_nowait((priority, message, meta))
 
+    async def start(self):
+        if not self._started:
+            self._started = True
+            for (
+                agent_creator,
+                container,
+                mirror_container_creator,
+            ) in self._agent_process_init_list:
+                await self.create_internal_agent_process(
+                    agent_creator, container, mirror_container_creator
+                )
```

Comment:
should be using `asyncio.gather` instead to decrease runtime

Reply:
gather did not work somehow, as the ProcessHandle is unhashable..
Using a poor men's version of gather now instead